config/config-package.py

```diff
@@ -32,6 +32,16 @@
 
 DOCKER_IMAGE = 'python:3.11-bullseye'
 
+GHA_DEFAULT_BRANCH = "main"
+GHA_DEFAULT_JOBS = [
+    "qa",
+    "test",
+    "coverage",
+    "dependencies",
+    "release-ready",
```

Comment:
If you are ok with renaming it to match the actual name:

```suggestion
    "release_ready",
```